jeni.py

```diff
@@ -399,6 +398,12 @@ class Injector(BaseInjector):
 
             with Injector() as injector:
                 injector.get('injector')
+
+        Local values can be provided when constructing the injector::
+
+            with RequestInjector(request=request) as injector:
+                injector.get('form:username')
```

Comment:
I'm sort of on the fence here.  The docs could use some more concrete examples that demonstrate useful patterns.  This hints at such an example, but it's obviously incomplete.